types/tableau-js-api/tableau-js-api-tests.ts

```diff
@@ -0,0 +1 @@
+import * as tableau from 'tableau-js-api';
```

Comment:
1. Can you add at least a simple usage example here?
2. Since the intended usage is in the browser, leave out the import statement and use `tableau.xxx` or `window.tableau.yyy` references.